packages/create-cloudflare/src/helpers/packages.ts

```diff
@@ -88,9 +88,10 @@ export async function getLatestPackageVersion(packageSpecifier: string) {
  */
 export const installWrangler = async () => {
 	const { npm } = detectPackageManager();
+	const wrangler = process.env.CI ? `wrangler@beta` : "wrangler@latest";
```

Comment:
Maybe try:

```suggestion
	const wrangler = process.env.CI ? `wrangler@beta --force` : "wrangler@latest";
```

Reply:
```
npm error Invalid tag name "beta --force" of package "wrangler@beta --force": Tags may not have any characters that encodeURIComponent encodes.
```

😢 unfortunately this breaks tests for all package managers